.github/workflows/test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2.1.5
       with:
-        go-version: '1.15'
+        go-version: '1.17'
```

Comment:
Lets do 1.18 since it's the latest, is that ok? Or do you have reservations there?

Reply:
Sounds good to me, I'll change this branch to bump to 1.18, merge and see how it impacts the acc-tests. I'm conscious that there are other steps in the workflow that might need bumping up, but I'll take it a step at a time

Reply:
Looks good!